bot/utils/checks.py

```diff
@@ -39,6 +39,7 @@ def in_whitelist_check(
     channels: Container[int] = (),
     categories: Container[int] = (),
     roles: Container[int] = (),
+    allow_dms: bool = False,
```

Comment:
I don't think this parameter was used anywhere.

Reply:
*play Jedi tricks* you didn't see anything